src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_MenuFlyout.cs

```diff
@@ -273,7 +273,7 @@ public async Task When_MenuFlyoutItem_CommandChanging()
 
 			await WindowHelper.WaitForLoaded(flyoutItem);
 
-			flyoutItem.InvokeClick();
+			flyoutItem.Invoke();
```

Comment:
Do we need additional manual testing for those?

Reply:
This should  be equivalent, the method was just named differently